### PR TITLE
Added ContractWrapper

### PR DESCRIPTION
**Pull request by aeharvlee**
## Proposed changes

This PR is for improving SDK usability of Contract Layer in [Common Architecture](https://kips.klaytn.com/KIPs/kip-34).

Improving SDK usability means providing users with a similar development experience
no matter what programming language the SDK is implemented in.

Each layer declared in Common Architecture should be accessed via `caver`.
Contract Layer in Common Architecture should be accessed via `caver.contract`.

In caver-js, creation of Contract instance is done by calling `caver.contract.create()`.

To make similar development experiences in caver-java, **I added a `ContractWrapper` class and add this as a member variable of Caver**

`ContractWrapper` wraps constructors of `Contract` class into `create` methods.
As adding this, we can create `Contract` instance by calling `caver.contract.create()` without passing `Caver` instance.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-java/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-java)
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by jimni1222 — APPROVED** [review]
LGTM =) !

File: core/src/main/java/com/klaytn/caver/contract/ContractWrapper.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ * 1. This class wraps constructors of Contract class
+ * 2. This class should be accessed via `caver.contract`
+ */
+public class ContractWrapper {
+    /**
+     * A Caver instance
+     */
+    private Caver caver;
+
+    public ContractWrapper(Caver caver) {
```
Comment (sirano11) [inline core/src/main/java/com/klaytn/caver/contract/ContractWrapper.java:34]: Please add a description.

Comment (aeharvlee) [reply core/src/main/java/com/klaytn/caver/contract/ContractWrapper.java:34]: @sirano11 
Oops... I missed it T_T
Thanks for your feedback. I'll add description.

Comment (aeharvlee) [reply core/src/main/java/com/klaytn/caver/contract/ContractWrapper.java:38]: I added a description with this [commit](https://github.com/klaytn/caver-java/pull/221/commits/42a2fbe5b1d57a6b38b88611b0336795818ed913).
Thanks, @sirano11 again.

File: core/src/main/java/com/klaytn/caver/Caver.java [hunk 2]
```diff
@@ -67,11 +68,16 @@
...
      * The AccountWrapper instance
      */
     public AccountWrapper account;
 
+    /**
+     * The ContractWrapper instance
+     */
+    public ContractWrapper contract;
+
     public Caver() {
```
Comment (sirano11) [inline core/src/main/java/com/klaytn/caver/Caver.java:81]: Please add a description Construct function

File: core/src/main/java/com/klaytn/caver/Caver.java [hunk 3]
```diff
@@ -67,11 +68,16 @@
...
         this(new HttpService(DEFAULT_URL));
```
Comment (sirano11) [inline core/src/main/java/com/klaytn/caver/Caver.java:82]: Please add a function description.

Comment (aeharvlee) [reply core/src/main/java/com/klaytn/caver/Caver.java:85]: @sirano11 
Thanks for your review :)

I added comment here [commit](https://github.com/klaytn/caver-java/pull/221/commits/6050503e450f8f9d35c6f4f31dc5547004c7e3b3).

Big thanks.

File: core/src/main/java/com/klaytn/caver/Caver.java [hunk 4]
```diff
@@ -67,11 +68,19 @@
...
     public IPFS ipfs;
 
-    /*
+    /**
      * The AccountWrapper instance
      */
     public AccountWrapper account;
 
+    /**
+     * The ContractWrapper instance
```
Comment (sirano11) [inline core/src/main/java/com/klaytn/caver/Caver.java:77]:
```suggestion
     * The ContractWrapper instance.<p>
     * It sets a HttpProvider that using DEFAULT_URL("http://localhost:8551").
```

**Review by sirano11 — APPROVED** [review]

**Review by sirano11 — APPROVED** [review]

**Pull request merged**